### PR TITLE
Fix the problems in the Header component

### LINKED ISSUE #174: Header is broken after token updates
**Description**
A) The `header.scss` file breaks after the token changes made in #161. Changes to spacing tokens, color tokens break the stylesheet. 

B) Simultaneously changes to the Logos story breaks the Header story too.

**Reproduction**
A) Clone the repository and build the stylesheets on the `master` branch at 35484bb with `npm run build`
B) Start storybook with `npm run storybook`

**Expected behavior**
A) Stylesheets should get compiled into a `vocabulary.css` file. 
B) Stortybook should start up with all contents.

**Cause**
Work done on PR #161 should have marked PR #145 as blocked. The order in which they were merged might have been a factor too.

### PR DESCRIPTION
**Pull request by dhruvkb**
**Fixes**

Fixes #174 by @dhruvkb 

**Description**

The PR fixes the broken references in `header.scss` and `header.stories.js` that were causing the build to fail.

**Type of PR** 
This PR is a hotfix.

**Technicalities**
Changes to Sass tokens and the logos story in #161 caused the header stylesheet and story to contain broken references, which in turn prevented the SCSS files from compiling and the storybook from building.

**Checklist:**
 
- [x] My pull request has a descriptive title (not a vague title like "Update `index.md`").
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

**Legal**
By submitting this PR, I agree to abide by the terms of the Developer 
Certificate of Origin.

<details>
<summary>Developer Certificate of Origin</summary>

Developer Certificate of Origin<br>
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.<br>
1 Letterman Drive<br>
Suite D4700<br>
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
</details>

### CODE REVIEW

**Review by hugosolar — COMMENTED** [review]
I can't see the arrow next to the items with dropdowns.
Also, I ask Chidi to remove the "Explore CC" button because it's parte from the global header component

**Comment by dhruvkb** [comment]
The existing arrow from Bulma looks like a chevron and is differen from the arrow used in Figma. That arrow needs to be imported from `creativecommons/fonts` for which we need to merge creativecommons/fonts#8 and resolve creativecommons/fonts#12.

**Comment by dhruvkb** [comment]
I'll remove the explore CC button. You're right that belongs to the superheader component.

**Comment by brenoferreira** [comment]
Screen Shot 2020-03-16 at 09 52 02
The problem with using a function to render the html is that the code docs displayed don't show what we need.

**Comment by brenoferreira** [comment]
Screen Shot 2020-03-16 at 09 53 55
some comments about the mobile layout

File: src/styles/header.scss [hunk 1]
```diff
@@ -1,81 +1,63 @@
...
-    background-color: $color-white;
+  background-color: $color-white;
 
-    .navbar-item:hover,
-    .navbar-link:hover,
-    .navbar-link:visited {
-      background-color: $color-white;
-      color: $color-black;
-    }
+  .container {
```
Comment (brenoferreira) [inline src/styles/header.scss]: Header with a `.container`? Why is this necessary?

File: src/stories/header.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false">
+            <span aria-hidden="true"></span>
+            <span aria-hidden="true"></span>
+            <span aria-hidden="true"></span>
+          </a>
+        </div>
+        <div class="navbar-menu">
+          <div class="navbar-end">
+            <div class="navbar-item has-dropdown is-hoverable">
+              <a class="navbar-link is-arrowless">Dropdown</a>
```
Comment (brenoferreira) [inline src/stories/header.stories.mdx]: This is supposed to have an arrow

Comment (dhruvkb) [reply src/stories/header.stories.mdx]: The arrow is waiting for creativecommons/fonts#8 and creativecommons/fonts#12.

Comment (dhruvkb) [reply src/stories/header.stories.mdx]: Added the arrow with 57b9c15.

File: src/styles/header.scss (hunk 1 above)
Comment (dhruvkb) [reply src/styles/header.scss]: After the arrival of PR #180, this is necessary for consistency with the footer.

**Comment by dhruvkb** [comment]
Using the `mdxSource` prop on the `Preview` component allows us to display any source code in the preview, so here it goes.
Screen Shot 2020-03-18 at 00 13 57
And the arrows too!
Screen Shot 2020-03-18 at 00 18 01

Just beautiful 😍!

**Comment by dhruvkb** [comment]
Blocked till #183 is resolved.

File: src/styles/header.scss (hunk 1 above)
Comment (hugosolar) [reply src/styles/header.scss]: In most of the websites we need to center the content of the header and the whole website, so `.container`. However I think we may need a header version without container for cc search right?

**Comment by dhruvkb** [comment]
@hugosolar @brenoferreira this is ready for merge.

**Review by hugosolar — APPROVED** [review]

**Review by brenoferreira — CHANGES_REQUESTED** [review]
The problems with the mobile hamburger menu still persist:

- it is not sticking to the right
- the menu doesn't open when you click it

**Comment by brenoferreira** [comment]
Screen Shot 2020-03-18 at 09 53 39

**Comment by brenoferreira** [comment]
I think you could try to implement something similar to how CC Search does the hamburger menu with a checkbox and CSS: https://github.com/creativecommons/cccatalog-frontend/blob/develop/src/components/NavSection.vue#L6

**Comment by brenoferreira** [comment]
About the checkbox to toggle the hamburger menu, I think that's going to be too hard to do because Bulma toggles visibility with the `.is-active` class. So I think that will require JS :(

For reference: https://bulma.io/documentation/components/navbar/#navbar-menu (See Javascript toggle)

**Review by brenoferreira — APPROVED** [review]

**Pull request merged**